CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+## How to Contribute
+
+We welcome contributions in the form of issues or pull requests!
+
+We want this to be a place where all are welcome to discuss and contribute, so please note that this project is released with a Contributor Code of Conduct. By participating in this project you agree to abide by its terms. Find the code of conduct in the ``CONDUCT.md`` file on GitHub.
```

Comment:
I think this could be simplified and that it's more likely people will read the code of conduct if you make it a hyperlink.

proposed revision:

```
We want this to be a place where all are welcome to discuss and contribute, so please note that by participating in this project you agree to abide by the terms of its [code of conduct](https://github.com/uptake/autofocus/blob/master/CONDUCT.md).
```

Reply:
Yeah, it's really wordy. Revised.